### DISCUSSION TITLE
Auto-sort Levels setting for profiles

### DISCUSSION THREAD

**Discussion by Arcadius2006** [LBPUnion/ProjectLighthouse#703] [Ideas]
This setting would arrange levels for you on your earth instead of having to place your levels manually, you could choose various sorting styles and then you would never have to think of where to place a level ever again

**Comment by vilijur** [comment]
That's an interesting concept. A few questions, if I may:

* What sorting styles will there be? Will there be Alphabetical? Publish Date? Most Plays?
* Where will levels be sorted? Will levels only be sorted on the site? Where would they go on the earth?
* What about multiple pages? How will auto sort manage with an earth that has levels that fill up multiple pages?
* What type of users do you think will benefit the most from this?

**Reply by Arcadius2006** [reply]
Sorting Order: Alphabetical, Publish date
Style: Circles, Columns, Rows, Polka Dot
Seperate LBP1/2/3 levels? (yes/no)
the sorting options are for the earth only, and don't affect the website (besides maybe the sorting order)

**Reply by Arcadius2006** [reply]
the type of user that would benefit the most from this is the usual archiver, who uploads dozens of levels a day and don't have time for finding the right place for a level

**Reply by vilijur** [reply]
I see..

**Reply by Arcadius2006** [reply]
you could also add another seperation option to seperate VS levels, cutscenes, CC levels, or move levels, and an option that combines both seperation types
